cluster/up.sh

```diff
@@ -29,6 +29,9 @@ if [[ "$KUBEVIRT_PROVIDER" =~ (ocp|okd)- ]]; then
     ./cluster/kubectl.sh delete ns openshift-cluster-samples-operator --wait=false
 fi
 
+# Install golang to run generate manifests
+sudo yum install -y golang
```

Comment:
That will be run on the machine where the tests are executed and if the tests are run locally, it will always ask for the root password (and then try to install golang tools unnecessarily). Do we need that?

Reply:
this is a valid question. if you run as a non-root/non-sudoer user - perhaps it is preferred to install go by downloading it
using wget & install it locally.

or is there an option to require the VM/OS to have go installed as a prerequisite ?

Reply:
For now we support only rpm based OSes. So I've added an `if` which check if `golang` is already installed, and skip the sudo install part if yes.